domain/miningmanager/mempool/config.go

```diff
@@ -0,0 +1,55 @@
+package mempool
+
+import (
+	"time"
+
+	"github.com/kaspanet/kaspad/util"
+
+	"github.com/kaspanet/kaspad/domain/dagconfig"
+)
+
+const (
+	defaultMaximumTransactionCount = 1_000_000
+
+	defaultTransactionExpireIntervalSeconds     uint64 = 60
+	defaultTransactionExpireScanIntervalSeconds uint64 = 10
+	defaultOrphanExpireIntervalSeconds          uint64 = 60
+	defaultOrphanExpireScanIntervalSeconds      uint64 = 10
+
+	defaultMaximumOrphanTransactionSize  = 100000
+	defaultMaximumOrphanTransactionCount = 50
+
+	defaultMinimumRelayTransactionFee = util.Amount(1000)
+)
+
+// Config represents a mempool configuration
+type Config struct {
+	MaximumTransactionCount               int
+	TransactionExpireIntervalDAAScore     uint64
+	TransactionExpireScanIntervalDAAScore uint64
+	OrphanExpireIntervalDAAScore          uint64
+	OrphanExpireScanIntervalDAAScore      uint64
+	MaximumOrphanTransactionSize          int
+	MaximumOrphanTransactionCount         int
```

Comment:
Is there a reason for the above two integers and MaximumTransactionCount to be signed? If not, please make them uints.

Reply:
Because they are being compared to `len(blabla)`, and len returns int.
So to save ourselves a cast, I'd rather leave those as ints.

LMK if you disagree.